src/certificates/certificate-details/CertificateDetails.jsx

```diff
@@ -58,21 +58,21 @@ const CertificateDetails = ({
     >
       <Stack>
         <Stack direction="horizontal" gap="1.5" className="certificate-details__info">
-          <p className="certificate-details__info__paragraph">
+          <p className="certificate-details__info-paragraph">
             <strong>{intl.formatMessage(messages.detailsCourseTitle)}:</strong> {detailsCourseTitle}
           </p>
-          <p className="certificate-details__info__paragraph--course-number">
+          <p className="certificate-details__info-paragraph--course-number">
```

Comment:
we use patern `block-block__element-element__modified-modifier`

`certificate-details__info-paragraph__course-number`